examples/spire/agent-daemonset.yaml

```diff
@@ -30,7 +31,7 @@ spec:
           args: ["-t", "30", "spire-server:8081"]
       containers:
         - name: spire-agent
-          image: gcr.io/spiffe-io/spire-agent:0.12.1
+          image: gcr.io/spiffe-io/spire-agent:1.0.2
```

Comment:
Should we update this in the separate PR?

Reply:
ok, done